### PR TITLE
[DOC] Add placeholder pages for intro

### PR DESCRIPTION
**Pull request by knylander-grafana**
Adds placeholder pages to use for introductory text. These pages are set to draft: true, so they won't render on the doc site until that is removed. 

This is the first step towards restructuring the content, per #2794 

Related: #2794

### CODE REVIEW

File: docs/sources/introduction/_index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: What is Pyroscope?
+menuTitle: What is Pyroscope?
+description: Learn about Pyrsocope and profiling.
+aliases:
```
Comment (knylander-grafana) [inline docs/sources/introduction/_index.md:5]:
```suggestion
```

File: docs/sources/introduction/_index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Sections
+
+<!-- Add sections replacing **Sections** with subtitles for each section. Concept topics or sections explain what and why. They do not explain how. If you are a new user, you might look for concept information to learn about what Grafana is, why it might be useful to you, and what the general workflow is. -->
+
+## Useful links
+
+<!-- Link out to task or reference topics related to the concept. -->
+
+- [Link example](https://grafana.com)
```
Comment (knylander-grafana) [inline docs/sources/introduction/_index.md:36]:
```suggestion
```

File: docs/sources/introduction/examples.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+title: Profiling examples
+menuTitle: Profiling examples
+description: What can you do with profiling data to resolve real-world problems?
+aliases:
```
Comment (knylander-grafana) [inline docs/sources/introduction/examples.md:5]:
```suggestion
```

File: docs/sources/introduction/pyroscope-in-grafana.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: Pyroscope and profiling in Grafana
+menuTitle: Pyroscope in Grafana
+description: Learn about how you can use profile data in Grafana.
+aliases:
```
Comment (knylander-grafana) [inline docs/sources/introduction/pyroscope-in-grafana.md:5]:
```suggestion
```

File: docs/sources/introduction/pyroscope-in-grafana.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## Sections
+
+<!-- Add sections replacing **Sections** with subtitles for each section. Concept topics or sections explain what and why. They do not explain how. If you are a new user, you might look for concept information to learn about what Grafana is, why it might be useful to you, and what the general workflow is. -->
+
+## Useful links
+
+<!-- Link out to task or reference topics related to the concept. -->
+
+- [Link example](https://grafana.com)
```
Comment (knylander-grafana) [inline docs/sources/introduction/pyroscope-in-grafana.md:37]:
```suggestion
```

**Review by Rperry2174 — COMMENTED** [review]
lpgtm!

**Comment by knylander-grafana** [comment]
> lpgtm!

@Rperry2174  Thanks for the review! It looks like you hit comment instead of approve. Would you be willing to hit that magic button?

**Review by Rperry2174 — APPROVED** [review]

**Pull request merged**